### PR TITLE
webui: tests: actually boot into the webui mode

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Review by VladimirSlavik — COMMENTED** [review]
Thank you, the most important part LGTM.

- Would it be useful to wrap our cockpit-ws in a systemd unit?
- The `wait(lambda: ...` part looks like it could be a function if we have it so many times.

**Comment by KKoukiou** [comment]
> Thank you, the most important part LGTM.
> 
>     * Would it be useful to wrap our `cockpit-ws` run in a systemd unit? That way we don't have to care about the actual command, the logs are preserved somewhere, and it's easy to do something like `systemctl <verb> anaconda-test-cockpit-ws`.

I have nothing against this approach, but let's please make it a different PR / task.
 
> 
>     * The `wait(lambda: ...` part looks like it could be a function if we have it so many times.

Will update.

**Review by VladimirSlavik — APPROVED** [review]
Thank you!

**Review by jkonecny12 — APPROVED** [review]
Looks good to me too.

**Comment by M4rtinK** [comment]
> I have nothing against this approach, but let's please make it a different PR / task.

Yeah, I like the suggestion but it can wait - lets note it down in an issue so it can be picked up later.

File: ui/webui/test/webuilib.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; If not, see <http://www.gnu.org/licenses/>.
+
+
+from testlib import wait  # pylint: disable=import-error
+
+
+def waitWebuiStarted(machine):
```
[the remark this answers is not in the document]
Comment (KKoukiou) [reply ui/webui/test/webuilib.py:22]: I guess you suggest the: s/waitWebuiStarted/wait_webui_started..
Btw is not the python 3 suggested string formatting method the f strings? https://github.com/rhinstaller/anaconda/blob/master/CONTRIBUTING.rst#code-conventions

**Review by poncovka — APPROVED** [review]

**Pull request merged**